<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Login</class>
 <widget class="QMainWindow" name="Form_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: rgb(0,0,0);
border-bottom : 2px solid rgba(255,255,255);
background-color: transparent;
font-size: 18px;
font-weight: 500;
color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color : rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color : rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: rgb(0,0,0);
border-bottom : 2px solid rgba(255,255,255);
background-color: transparent;
font-size: 18px;
font-weight: 500;
color: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 255, 255);
color: black;
font-size: 16px;
font-family: &quot;Poppins&quot;;
border-radius: 15px;
font-weight: 600;
</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-9</y>
      <width>1091</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,220);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: 500;
font-size: 18px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_nickname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color : rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Nickname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_nickname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: rgb(0,0,0);
border-bottom : 2px solid rgba(255,255,255);
background-color: transparent;
font-size: 18px;
font-weight: 500;
color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lastname">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color : rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Lastname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_lastname">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: rgb(0,0,0);
border-bottom : 2px solid rgba(255,255,255);
background-color: transparent;
font-size: 18px;
font-weight: 500;
color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;
font-weight: 600;
font-size : 20px</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;
font-weight: 600;
font-size : 20px</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;
font-weight: 600;
font-size : 20px</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;
font-weight: 600;
font-size : 20px</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font-size: 16px;
font-weight: 600;
color: white;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font-size: 16px;
font-weight: bold;
color: white;</string>
    </property>
    <property name="text">
     <string>Sudah punya akun ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color : rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: rgb(0,0,0);
border-bottom : 2px solid rgba(255,255,255);
background-color: transparent;
font-size: 18px;
font-weight: 500;
color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;
font-weight: 600;
font-size : 20px</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>601</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-image: url(:/registerscreen/img/bg-register.jpeg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logo/img/logo.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;
color: yellow;
font-weight : 500;</string>
    </property>
    <property name="text">
     <string>Warning</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>710</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
color: white;
font-weight : 500;</string>
    </property>
    <property name="text">
     <string>*Isi semua kolom yang tersedia </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>730</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
color: white;
font-weight : 500;</string>
    </property>
    <property name="text">
     <string>*Semua data dipertanggungjawabkan dengan baik</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>input_username</zorder>
   <zorder>label_username</zorder>
   <zorder>label_password</zorder>
   <zorder>input_password</zorder>
   <zorder>btn_register</zorder>
   <zorder>label_error</zorder>
   <zorder>label_nickname</zorder>
   <zorder>input_nickname</zorder>
   <zorder>label_lastname</zorder>
   <zorder>input_lastname</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>btn_login</zorder>
   <zorder>label</zorder>
   <zorder>label_username_2</zorder>
   <zorder>input_email</zorder>
   <zorder>label_6</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>widget</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../util/util.qrc"/>
 </resources>
 <connections/>
</ui>
